plugins/scaffolder-backend/src/scaffolder/tasks/NunjucksWorkflowRunner.ts

```diff
@@ -100,26 +102,43 @@ const createStepLogger = ({
   task: TaskContext;
   step: TaskStep;
 }) => {
-  const metadata = { stepId: step.id };
-  const taskLogger = winston.createLogger({
+  const stepLogStream = new PassThrough();
+  stepLogStream.on('data', async data => {
+    const message = data.toString().trim();
+    if (message?.length > 1) {
+      await task.emitLog(message, { stepId: step.id });
+    }
+  });
+
+  const taskLogger = WinstonLogger.create({
     level: process.env.LOG_LEVEL || 'info',
```

Comment:
Probably unfortunate to have this env var affect the client logs

Reply:
Yeah most probably, it's unfortunate that the loggers are both shared, otherwise it might make sense that the frontend only gets `> INFO` and the backend could list `DEBUG` if that was enabled with the `process.env.LOG_LEVEL`